.gitignore

```diff
@@ -2,3 +2,5 @@
 0.data-download/data/*
 1.data-exploration/test_data_loader.ipynb
 __pycache__
+2.train-VAE/hyper_parameterization.ipynb
```

Comment:
is this intentional? Should we include this file? Oh, is it actually saved later as a `.py` file (instead of `.ipynb`)?

Reply:
this file was created as an outline for hyper parameterization but had not uses. i should have deleted it instead of using gitignore